### PR TITLE
Variety Pizza Crates

### PR DESCRIPTION
**Pull request by flowercuco**
## About The Pull Request
hhostTnrjf

Adds three new pizza crates that are orderable via cargo's supply shuttle!
Mothic Pizza crates containing mothic marhgerita, firecracker, five cheese, pesto, and white pie!

Flatbread Pizza crates containing one imperial flatbread, and two each of rustic and italic flatbreads!

The Ultimate Universal Pizza Crate that contains a whopping ten pizzas for stomachs of all kinds! Margherita, Mushroom, Meat, Pineapple, Mothic firecracker, pesto, and five cheese pizzas, then all three current lizard flatbreads.

dreamseeker_Xzal9tHD9R

This should also add this to the services order console! Free pizza!

## Why It's Good For The Game

People wanted this and thought it would be a good idea and it means that gimmick chefs can focus on doing their fun gimmicks without worrying that if they forgot to/can't incorporate certain palettes it is not to the detriment of other players.

## Changelog

:cl:
add: three new pizza crates in the cargo shuttle
/:cl:

### CODE REVIEW

File: orbstation/modules/cargo/orders.dm [hunk 1]
```diff
@@ -3,3 +3,48 @@
 	desc = "Contains two Ham Helmets, one for you, and one for your beloved, be they a person or a corgi."
 	cost = PAYCHECK_CREW * 2
 	contains = list(/obj/item/clothing/head/helmet/ham,/obj/item/clothing/head/helmet/ham)
+
+/datum/supply_pack/organic/flatbread
+	name = "Flatbread Crate"
+	desc = "Gluten-Free! A taste of home for Tizarian's everywhere. \
```
Comment (skylord-a52) [inline orbstation/modules/cargo/orders.dm:9]: Should be "Tizirans" (this spelling, no apostrophe)

File: orbstation/modules/cargo/orders.dm [hunk 2]
```diff
@@ -3,3 +3,48 @@
...
+					/obj/item/pizzabox/rustic_flatbread,
+					/obj/item/pizzabox/italic_flatbread,
+					/obj/item/pizzabox/rustic_flatbread,
+					/obj/item/pizzabox/italic_flatbread,
+				)
+	crate_name = "flatbread crate"
+
+/datum/supply_pack/organic/mothicpizza
+	name = "Mothic Pizza Crate"
+	desc = "You'll be drawn to these Pizzas as if they were a flame! \
```
Comment (skylord-a52) [inline orbstation/modules/cargo/orders.dm:22]: I think this description could be better... but I can't think of anything better myself, so I probably shouldn't be throwing stones.

**Review by skylord-a52 — CHANGES_REQUESTED** [review]

File: orbstation/modules/cargo/orders.dm (hunk 1 above)
Comment (flowercuco) [reply orbstation/modules/cargo/orders.dm]: fixed!

File: orbstation/modules/cargo/orders.dm (hunk 2 above)
Comment (flowercuco) [reply orbstation/modules/cargo/orders.dm]: i think the lizard one is going to be fine but the moth one is still eluding me yeah

**Review by skylord-a52 — APPROVED** [review]

**Pull request merged**